libs/orders/src/lib/components/order-list-manager/order-list-manager.tsx

```diff
@@ -198,8 +184,34 @@ export const OrderListManager = ({
           }}
           order={editOrder}
           onSubmit={(fields) => {
+            if (!editOrder.market) {
+              return;
+            }
+            create({
+              orderAmendment: {
```

Comment:
suggestion (non blocking): Might be good to add a normaliseOrderAmendment function, similar for normaliseOrderSubmission, that could be easily unit tested.